build/yamls/base/conf/antrea-agent.conf

```diff
@@ -5,6 +5,10 @@ featureGates:
 # Service traffic.
 #  AntreaProxy: true
 
+# Enable full Service support in AntreaProxy in antrea-agent. All type of Services can be
+# accessed from outside the cluster.
+#  AntreaProxyFull: false
```

Comment:
@jianjuns @tnqn do you think we should have a discussion about whether a Feature Gate is the appropriate toggle mechanism here? Unless I am mistaken, a Feature Gate is meant to be removed eventually (feature goes to GA, or is removed altogether). Unless you think that that NodePort / LoadBalancer support should always be enabled by default eventually, maybe we should consider a configuration parameter?

Reply:
Do not have a good sense a Feature should be always enabled finally. But do you think we can use a Feature Gate to isolate the change, and change it to a config option later?

Reply:
A Feature Gate is meant to go to GA, and then cannot be disabled any more. If we intend to have a config parameter eventually, it kind of feels like we could have it from the get-go (disabled by default at first) and avoid using a Feature Gate altogether, which are typically used to gate an API (?). I can't think of an example in K8s where you would have both a Feature Gate and (either simultaneously or when the feature goes to GA) a config parameter. Maybe @tnqn can think of one?

Reply:
Yeah, I got the point. But my question is do we need a mechanism to isolate code changes and indicate the phase (alpha, beta, etc.) of a feature?

Reply:
Thinking about this more, having both can make sense to isolate code changes, but we can have both from the get go. Maybe the CPUManager (https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/) feature gate is a good comparison, as it still requires a flag `--cpu-manager-policy` to affect scheduling from what I can tell.